libs/scully/src/lib/utils/handlers/defaultAction.ts

```diff
@@ -40,7 +40,7 @@ async function plugin(localBaseFilter = baseFilter): Promise<HandledRoute[]> {
     const processedRoutes = await findPlugin(processRoutes)(handledRoutes);
 
     /** handleRouteDiscoveryDone run the discoverydone plugins */
-    const discoveryDone = handleRouteDiscoveryDone(processedRoutes);
+    const discoveryDone = await handleRouteDiscoveryDone(processedRoutes);
```

Comment:
This await must not be there. We don't need to `await` for this to complete.
if you want to reuse the results use `processedRoutes` plugins.

Reply:
Sorry about that, didn't mean for that to be committed.